components/notification/index.en-US.md

```diff
@@ -0,0 +1,56 @@
+---
+category: Components
+type: Views
+noinstant: true
+english: Notification
+---
+
+To display a notification message globally.
+
+## When to use
+To display a notification message at the top right of the view port. Typically it can be
+used in the following case:
+
+- A notification with complext content.
+- A notification of interaction with users, typically, it will provide next step hint
+for users.
+- A notification is pushed by application.
+
+## API
+
+- `notification.success(config)`
+- `notification.error(config)`
+- `notification.info(config)`
+- `notification.warning(config)`
+- `notification.warn(config)`
+- `notification.close(key: String)`
+- `notification.destroy()`
+
+The properties of config are as follows:
+
+| Property   | Description                                     | Type         | Default |
+|----------- |---------------------------------------------    | ----------- |--------|
+| message    | The title of notification box (required)        | React.Element or String      | No     |
+| description | The content of notification box (required)     | React.Element or String      | No     |
+| btn        | Custom close button                                   | React.Element      | No     |
+| key        | The unique identity of current notification                                 | String      | No     |
+| onClose    | Specify a function that will be called after clicking the default close button  | Function    | No     |
+| duration   | A notification box is closed after 4.5s by default, but when you specify `duration` to null, it will never be closed automatically | Number    | 4.5     |
```

Comment:
| duration   | A notification box is closed after 4.5s by default. When specifying `duration` to null or `0`, it will never be closed automatically. | Number    | 4.5     |